onnxruntime/core/providers/qnn/builder/qnn_node_group/lpbqgemm_fusion.h

```diff
@@ -46,7 +46,9 @@ class LowPowerBlockQuantizedGemmFusion : public IQnnNodeGroup {
 
  private:
   std::array<const NodeUnit*, 6> node_units_;
+  mutable std::vector<const NodeUnit*> filtered_node_units_;
+  mutable bool filtered_node_units_initialized_ = false;
```

Comment:
can these not be mutable? it seems that `filtered_node_units_` could be populated in the ctor.